src/lib/djerba/plugins/supplement/plugin.py

```diff
@@ -10,21 +10,33 @@ class main(plugin_base):
     DEFAULT_CONFIG_PRIORITY = 1000
     MAKO_TEMPLATE_NAME = 'supplementary_materials_template.html'
     SUPPLEMENT_DJERBA_VERSION = 0.1
-
+    INPUT_PARAMS_FILE = "input_params.json"
     ASSAY = "assay"
     
     def specify_params(self):
-        required = [
+        discovered = [
             self.ASSAY
         ]
-        for key in required:
-            self.add_ini_required(key)
+        for key in discovered:
+            self.add_ini_discovered(key)
         self.set_ini_default(core_constants.ATTRIBUTES, 'clinical')
 
     def configure(self, config):
         config = self.apply_defaults(config)
         wrapper = self.get_config_wrapper(config)
         wrapper.set_my_priorities(self.DEFAULT_CONFIG_PRIORITY)
+        
+        # If input_params.json exists, read it
+        work_dir = self.workspace.get_work_dir()
```

Comment:
The workspace class has methods to check if files exist -- preferable to use these